nilearn/_utils/compat.py

```diff
@@ -43,3 +49,8 @@ class _urllib():
         request = module_lookup
         error = module_lookup
         parse = module_lookup
+
+    def md5_hash(string):
```

Comment:
Is there no way to combine the two md5_hash() functions with an option for UNICODE or not?